Sources/ProjectDescription/Environment.swift

```diff
@@ -5,6 +5,16 @@ public struct Environment {
     public enum Value {
         case boolean(Bool)
         case string(String)
+
+        public func getString(default defaultString: String = "") -> String {
+            if case let .string(value) = self { return value }
+            return defaultString
+        }
+
+        public func getBoolean() -> Bool {
```

Comment:
I'd also expose `default:` here as you did with `getString`.

Reply:
No worries this has been updated